### PR TITLE
mrc-1506: mock mintr api

### PR DESCRIPTION
**Pull request by richfitz**
This PR builds out the spec, currently just returning mock data (i.e., the same data in the spec examples).  This validates the schema to a degree, but there are likely a ton of edge cases missing, especially as this is the first thing using pkgapi...

For the test version, don't use the instructions in the readme, do

```
docker run --rm -p 8888:8888 mrcide/mintr:mrc-1506
```

### CODE REVIEW

File: tests/testthat/test-api-endpoints.R [hunk 1]
```diff
@@ -0,0 +1,75 @@
+context("api: endpoints")
+
+
+test_that("graph prevalence config", {
+  res <- target_graph_prevalence_config()
+  expect_is(res, "json")
```
Comment (richfitz) [inline tests/testthat/test-api-endpoints.R:6]: Note that nothing here really constitutes a test of what is returned!
